cloud/server-scaffolding/src/main/java/org/webpieces/microsvc/server/api/FilterConfig.java

```diff
@@ -11,8 +12,9 @@ public class FilterConfig {
 
     private boolean enableErrorHandling = true;
 
-    public FilterConfig(String packageRegEx) {
```

Comment:
FYI, the original constructor is already in use [here](https://github.com/deanhiller/webpieces/blob/master/webserver-templates/webpiecesServerBuilder/TemplateJsonMicroSvc/production/src/main/java/webpiecesxxxxxpackage/base/ProdServerMeta.java) line62 . I've changed it but not sure if it impacts that usage.